packages/cli/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "preinstall": "node ./scripts/preinstall.js",
-    "test": "jest --env node --verbose --runInBand --bail --forceExit",
+    "test": "jest --env node --verbose --runInBand --bail",
```

Comment:
Removed `--forceExit` because the update notifier is async and the process will out live the test and causes jest to complain.